content/en/docs/additional-concepts/sidecar-containers/_index.md

```diff
@@ -25,28 +25,23 @@ In a sidecar pattern, the functionality of the main container is extended or enh
 In {{<link "persistent-storage">}} you created a MariaDB deployment. In this task you are going to add the [Prometheus MySQL exporter](https://github.com/prometheus/mysqld_exporter) to it.
 
 {{% onlyWhenNot openshift %}}
```

Comment:
shall I change this also for OpenShift?

Reply:
What exactly?

Reply:
Ah, in the overview it didn't show the following changes. We can also change this in OpenShift, that's ok, thanks.

Reply:
And this is also changed in the oc variants